pkg/apis/ceph.rook.io/v1beta1/types.go

```diff
@@ -170,13 +170,16 @@ type FilesystemList struct {
 // FilesystemSpec represents the spec of a file system
 type FilesystemSpec struct {
 	// The metadata pool settings
-	MetadataPool PoolSpec `json:"metadataPool"`
+	MetadataPool PoolSpec `json:"metadataPool,omitempty"`
 
 	// The data pool settings
-	DataPools []PoolSpec `json:"dataPools"`
+	DataPools []PoolSpec `json:"dataPools,omitempty"`
```

Comment:
I think this would be a non-breaking change from v0.8, correct?

Reply:
Yep, the same yaml file should always work before or after the change